cloudsmith/README.md

```diff
@@ -0,0 +1,64 @@
+# Agent Check: Cloudsmith
+
+## Overview
+
+This check monitors [Cloudsmith][1] through the Datadog Agent.
+- Monitor storage, banwidth and token usage in your Cloudsmith account. 
+
+
+## Setup
+
+The Cloudsmith check is not included in the [Datadog Agent][2] package, so you need to install it.
+
+### Installation
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Cloudsmith check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
+
+1. Run the following command to install the Agent integration:
+
+   ```shell
+   datadog-agent integration install -t datadog-cloudsmith==<INTEGRATION_VERSION>
+   `3 backticks here`
+
+2. Configure your integration similar to core [integrations][4].
+
+
+[2]: https://app.datadoghq.com/account/settings#agent
+[3]: https://docs.datadoghq.com/agent/guide/use-community-integrations/
+[4]: https://docs.datadoghq.com/getting_started/integrations/
```

Comment:
These references should go at the bottom and the other references will need to be re-numbered.

Reply:
Sorry about that Ruth- I tested that out now and all the links should be correct.